source/_integrations/emoncms.markdown

```diff
@@ -19,177 +19,6 @@ related:
-or the feed ID otherwise, and will be prefixed with "Emoncms", e.g., "Emoncms Total Power" or "Emoncms Feed 5".
-If the `id` property is anything but `1`, the ID will be shown as well, e.g., "Emoncms 2 Feed 5".
-
-If `sensor_names` is used, any feeds with defined names will get those names exactly, with no prefix.
-
-### Examples
-
-In this section you find some more examples of how this sensor can be used.
-
-Minimal configuration. All feeds are added as sensors with the unit of measurement being set by the Emoncms Feed or the default unit.
-
-```yaml
-sensor:
-  - platform: emoncms
-    api_key: API_KEY
-    url: https://emoncms.org
-    id: 1
-```
-
-Display only feeds with their feed IDs specified in `include_only_feed_id`.
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: emoncms
-    api_key: API_KEY
-    url: https://emoncms.org
-    id: 1
-    unit_of_measurement: "W"
-    include_only_feed_id:
-      - 107
-      - 105
-```
-
-Display all feeds except feeds with their feed ID specified in `exclude_feed_id`.
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: emoncms
-    api_key: API_KEY
-    url: https://emoncms.org
-    id: 1
-    unit_of_measurement: "kWh"
-    exclude_feed_id:
-      - 107
-      - 105
-```
-
-Display only feeds with their feed IDs specified in `include_only_feed_id` and give the feed sensors a name using `sensor_names`. You don't have to specify all feeds names in `sensor_names`, the remaining sensor names will be chosen based on `id` and the Emoncms `feedid`.
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: emoncms
-    api_key: API_KEY
-    url: https://emoncms.org
-    id: 1
-    unit_of_measurement: "kW"
-    include_only_feed_id:
-      - 5
-      - 120
-    sensor_names:
-      5: "feed 1"
-      48: "kWh feed"
-      61: "amp feed"
-      110: "watt feed"
-```
-
-Use a `value_template` to add 1500 to the feed value for all specified feed IDs in `include_feed_id`.
-
-{% raw %}
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: emoncms
-    api_key: API_KEY
-    url: https://emoncms.org
-    scan_interval: 15
-    id: 1
-    value_template: "{{ value | float + 1500 }}"
-    include_only_feed_id:
-      - 107
-      - 106
-```
-
-{% endraw %}
-
-Display feeds from the same Emoncms instance with 2 groups of feeds, where one has a differing `scan_interval` and the other a differing `unit_of_measurement`.
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: emoncms
-    api_key: API_KEY
-    url: https://emoncms.org
-    scan_interval: 30
-    id: 1
-    unit_of_measurement: "W"
-    include_only_feed_id:
-      - 107
-      - 106
-  - platform: emoncms
-    api_key:  API_KEY
-    url: https://emoncms.org
-    id: 2
-    scan_interval: 60
-    unit_of_measurement: "A"
-    include_only_feed_id:
-      - 108
-      - 61
-```
+{% include integrations/config_flow.md %}
```

Comment:
Below the config flow, please add a section called Sensors, listing all the sensors with name, unit, description and any other helpful information. So that we don't loose this information.

Reply:
done